torch/csrc/api/include/torch/nn/modules/linear.h

```diff
@@ -66,6 +66,32 @@ TORCH_MODULE(Linear);
 
 // ============================================================================
 
+/// A placeholder for Flatten operator
+class TORCH_API FlattenImpl : public Cloneable<FlattenImpl> {
+ public:
+  FlattenImpl(int64_t start_dim, int64_t end_dim) : FlattenImpl(FlattenOptions(start_dim, end_dim)) {}
```

Comment:
We can remove this constructor and only keep `explicit FlattenImpl(const FlattenOptions& options_);`